tests/test_store.py

```diff
@@ -92,9 +93,14 @@ def test_init(self):
 
 
 class TestSynapseStorage:
-    def test_init(self, synapse_store):
+    def test_init(self, synapse_store:SynapseStorage) -> None:
         assert synapse_store.storageFileview == "syn23643253"
         assert isinstance(synapse_store.storageFileviewTable, pd.DataFrame)
+        assert synapse_store.root_synapse_cache == ".synapseCache"
+
+    def test__purge_synapse_cache(self, synapse_store:SynapseStorage) -> None:
```

Comment:
I think this is testing if the `purge_synapse_cache` function can be called. But this is not testing if the `purge_synapse_cache`  function is working as expected. Could you add a test to test if `_purge_synapse_cache` is working as expected?